app/javascript/graph_builder.js

```diff
@@ -19,11 +17,9 @@ document.addEventListener('turbolinks:load', function() {
   }).then(function(run) {
     document.getElementById('graph-spinner').hidden = true
     document.getElementById('graph-holder').hidden = false
-    build_pb_graph(run.run)
     if (run.run.program === 'livesplit') {
```

Comment:
Completely unrelated, but technically other timers could supply this data now through the exchange format. I'd like developers of new timers to get the full suite of capabilities without first having to ask us for permission, is there a simple way we can change this check to "has segment histories"?